stores/filestore.go

```diff
@@ -3893,6 +3871,12 @@ func (fs *FileStore) newFileSubStore(channel string, limits *SubStoreLimits, doR
 	} else {
 		fs.fm.unlockFile(ss.file)
 	}
+	if fs.opts.AutoSync > 0 {
+		ss.Lock()
+		ss.allDone.Add(1)
+		ss.syncTimer = time.AfterFunc(fs.opts.AutoSync, ss.autoSync)
```

Comment:
What if already set?

Reply:
Can't. This is the creation of the substore. For a given substore, this happen only once.

Reply:
ok